### ISSUE TITLE
BatchNorm3d cannot be converted

### ISSUE THREAD

**Issue by DanLuoNEU** [NVIDIA-AI-IOT/torch2trt#725]
Hi,

Although after installation, I can see BatchNorm3d is included in the converters, but there is no such converter under torch2trt.converters or troch2trt.convert_BatchNorm3d

```bash
running install
/home/dluo/anaconda3/envs/py37pt1102/lib/python3.7/site-packages/setuptools/command/install.py:37: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  setuptools.SetuptoolsDeprecationWarning,
/home/dluo/anaconda3/envs/py37pt1102/lib/python3.7/site-packages/setuptools/command/easy_install.py:147: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
  EasyInstallDeprecationWarning,
running bdist_egg
running egg_info
writing torch2trt.egg-info/PKG-INFO
writing dependency_links to torch2trt.egg-info/dependency_links.txt
writing top-level names to torch2trt.egg-info/top_level.txt
reading manifest file 'torch2trt.egg-info/SOURCES.txt'
adding license file 'LICENSE.md'
writing manifest file 'torch2trt.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
running build_py
creating build/bdist.linux-x86_64/egg
creating build/bdist.linux-x86_64/egg/torch2trt
copying build/lib/torch2trt/__init__.py -> build/bdist.linux-x86_64/egg/torch2trt
copying build/lib/torch2trt/utils.py -> build/bdist.linux-x86_64/egg/torch2trt
copying build/lib/torch2trt/calibration.py -> build/bdist.linux-x86_64/egg/torch2trt
copying build/lib/torch2trt/test.py -> build/bdist.linux-x86_64/egg/torch2trt
creating build/bdist.linux-x86_64/egg/torch2trt/tests
copying build/lib/torch2trt/tests/__init__.py -> build/bdist.linux-x86_64/egg/torch2trt/tests
copying build/lib/torch2trt/tests/test_contiguous.py -> build/bdist.linux-x86_64/egg/torch2trt/tests
creating build/bdist.linux-x86_64/egg/torch2trt/tests/torchvision
copying build/lib/torch2trt/tests/torchvision/classification.py -> build/bdist.linux-x86_64/egg/torch2trt/tests/torchvision
copying build/lib/torch2trt/tests/torchvision/save_load.py -> build/bdist.linux-x86_64/egg/torch2trt/tests/torchvision
copying build/lib/torch2trt/tests/torchvision/__init__.py -> build/bdist.linux-x86_64/egg/torch2trt/tests/torchvision
copying build/lib/torch2trt/tests/torchvision/segmentation.py -> build/bdist.linux-x86_64/egg/torch2trt/tests/torchvision
copying build/lib/torch2trt/module_test.py -> build/bdist.linux-x86_64/egg/torch2trt
creating build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/layer_norm.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/BatchNorm1d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/Linear.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/clamp.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/Conv.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/adaptive_max_pool2d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/batch_norm.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/floordiv.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/dummy_converters.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/getitem.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/AdaptiveAvgPool3d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/add.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/tensor.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/normalize.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/div.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/expand.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/split.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/max_pool1d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/__init__.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/min.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/LogSoftmax.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/adaptive_avg_pool3d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/sigmoid.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/group_norm.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/adaptive_avg_pool2d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/permute.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/relu.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/softmax.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/cat.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/prod.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/compare.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/conv_functional.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/avg_pool.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/mean.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/mul.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/max_pool2d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/relu6.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/roll.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/stack.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/AdaptiveAvgPool2d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/gelu.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/narrow.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/transpose.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/sub.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/pow.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/unary.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/ne.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/BatchNorm3d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/silu.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/prelu.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/tanh.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/instance_norm.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/adaptive_max_pool3d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/identity.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/ConvTranspose.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/Conv1d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/chunk.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/max.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/mod.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/view.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/activation.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/ConvTranspose2d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/sum.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/pad.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/BatchNorm2d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/Conv2d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/max_pool3d.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/interpolate.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/converters/clone.py -> build/bdist.linux-x86_64/egg/torch2trt/converters
copying build/lib/torch2trt/torch2trt.py -> build/bdist.linux-x86_64/egg/torch2trt
creating build/bdist.linux-x86_64/egg/torch2trt/contrib
copying build/lib/torch2trt/contrib/__init__.py -> build/bdist.linux-x86_64/egg/torch2trt/contrib
creating build/bdist.linux-x86_64/egg/torch2trt/contrib/qat
creating build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/layers
copying build/lib/torch2trt/contrib/qat/layers/quant_conv.py -> build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/layers
copying build/lib/torch2trt/contrib/qat/layers/_utils.py -> build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/layers
copying build/lib/torch2trt/contrib/qat/layers/__init__.py -> build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/layers
copying build/lib/torch2trt/contrib/qat/layers/quant_activation.py -> build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/layers
copying build/lib/torch2trt/contrib/qat/__init__.py -> build/bdist.linux-x86_64/egg/torch2trt/contrib/qat
creating build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/converters
copying build/lib/torch2trt/contrib/qat/converters/__init__.py -> build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/converters
copying build/lib/torch2trt/contrib/qat/converters/QuantRelu.py -> build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/converters
copying build/lib/torch2trt/contrib/qat/converters/QuantConv.py -> build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/converters
copying build/lib/torch2trt/contrib/qat/converters/QuantConvBN.py -> build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/converters
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/utils.py to utils.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/calibration.py to calibration.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/test.py to test.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/tests/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/tests/test_contiguous.py to test_contiguous.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/tests/torchvision/classification.py to classification.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/tests/torchvision/save_load.py to save_load.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/tests/torchvision/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/tests/torchvision/segmentation.py to segmentation.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/module_test.py to module_test.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/layer_norm.py to layer_norm.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/BatchNorm1d.py to BatchNorm1d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/Linear.py to Linear.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/clamp.py to clamp.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/Conv.py to Conv.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/adaptive_max_pool2d.py to adaptive_max_pool2d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/batch_norm.py to batch_norm.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/floordiv.py to floordiv.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/dummy_converters.py to dummy_converters.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/getitem.py to getitem.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/AdaptiveAvgPool3d.py to AdaptiveAvgPool3d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/add.py to add.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/tensor.py to tensor.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/normalize.py to normalize.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/div.py to div.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/expand.py to expand.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/split.py to split.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/max_pool1d.py to max_pool1d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/min.py to min.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/LogSoftmax.py to LogSoftmax.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/adaptive_avg_pool3d.py to adaptive_avg_pool3d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/sigmoid.py to sigmoid.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/group_norm.py to group_norm.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/adaptive_avg_pool2d.py to adaptive_avg_pool2d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/permute.py to permute.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/relu.py to relu.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/softmax.py to softmax.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/cat.py to cat.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/prod.py to prod.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/compare.py to compare.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/conv_functional.py to conv_functional.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/avg_pool.py to avg_pool.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/mean.py to mean.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/mul.py to mul.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/max_pool2d.py to max_pool2d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/relu6.py to relu6.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/roll.py to roll.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/stack.py to stack.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/AdaptiveAvgPool2d.py to AdaptiveAvgPool2d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/gelu.py to gelu.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/narrow.py to narrow.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/transpose.py to transpose.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/sub.py to sub.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/pow.py to pow.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/unary.py to unary.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/ne.py to ne.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/BatchNorm3d.py to BatchNorm3d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/silu.py to silu.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/prelu.py to prelu.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/tanh.py to tanh.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/instance_norm.py to instance_norm.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/adaptive_max_pool3d.py to adaptive_max_pool3d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/identity.py to identity.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/ConvTranspose.py to ConvTranspose.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/Conv1d.py to Conv1d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/chunk.py to chunk.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/max.py to max.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/mod.py to mod.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/view.py to view.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/activation.py to activation.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/ConvTranspose2d.py to ConvTranspose2d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/sum.py to sum.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/pad.py to pad.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/BatchNorm2d.py to BatchNorm2d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/Conv2d.py to Conv2d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/max_pool3d.py to max_pool3d.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/interpolate.py to interpolate.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/converters/clone.py to clone.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/torch2trt.py to torch2trt.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/contrib/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/layers/quant_conv.py to quant_conv.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/layers/_utils.py to _utils.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/layers/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/layers/quant_activation.py to quant_activation.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/converters/__init__.py to __init__.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/converters/QuantRelu.py to QuantRelu.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/converters/QuantConv.py to QuantConv.cpython-37.pyc
byte-compiling build/bdist.linux-x86_64/egg/torch2trt/contrib/qat/converters/QuantConvBN.py to QuantConvBN.cpython-37.pyc
creating build/bdist.linux-x86_64/egg/EGG-INFO
copying torch2trt.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
copying torch2trt.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying torch2trt.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying torch2trt.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
torch2trt.contrib.qat.layers.__pycache__._utils.cpython-37: module MAY be using inspect.stack
creating 'dist/torch2trt-0.3.0-py3.7.egg' and adding 'build/bdist.linux-x86_64/egg' to it
removing 'build/bdist.linux-x86_64/egg' (and everything under it)
Processing torch2trt-0.3.0-py3.7.egg
removing '/home/dluo/anaconda3/envs/py37pt1102/lib/python3.7/site-packages/torch2trt-0.3.0-py3.7.egg' (and everything under it)
creating /home/dluo/anaconda3/envs/py37pt1102/lib/python3.7/site-packages/torch2trt-0.3.0-py3.7.egg
Extracting torch2trt-0.3.0-py3.7.egg to /home/dluo/anaconda3/envs/py37pt1102/lib/python3.7/site-packages
Removing torch2trt 0.3.0 from easy-install.pth file
Adding torch2trt 0.3.0 to easy-install.pth file

Installed /home/dluo/anaconda3/envs/py37pt1102/lib/python3.7/site-packages/torch2trt-0.3.0-py3.7.egg
Processing dependencies for torch2trt==0.3.0
Finished processing dependencies for torch2trt==0.3.0
```

BatchNorm* Modules under torch2trt.converters
![image](https://user-images.githubusercontent.com/56852645/165004921-ec703ca2-9c4f-4863-a18a-439d3b07eb93.png)

Modules under torch2trt.convert_*
![image](https://user-images.githubusercontent.com/56852645/165004818-75f3c133-52ea-4845-9da0-be5cc7340b11.png)

Environment:
```
PyTorch - 1.10.2
torch2trt - 0.3.0-py3.7
CUDA version -  11.6 
torch._C._cuda_getCompiledVersion() - 11030 
```

**Comment by jaybdub** [comment]
Hi @DanLuoNEU ,

Thanks for pointing this out!

It looks like the import is just missing from this file [this file](https://github.com/NVIDIA-AI-IOT/torch2trt/blob/458394febf25b879cb6ea2e12f0060ef4beae7a3/torch2trt/converters/__init__.py#L9)

I'll add the import and a test case and report back.

Best,
John

**Comment by jaybdub** [comment]
Now fixed in master via this PR https://github.com/NVIDIA-AI-IOT/torch2trt/pull/730.

Hope this helps, let me know if you run into other issues.

Best,
John

**Comment by DanLuoNEU** [comment]
> Now fixed in master via this PR #730.
> 
> Hope this helps, let me know if you run into other issues.
> 
> Best, John

Hi John,

Thanks for the in-time fixing! I can see it works for BN3d!

Best,
Dan
